src/clients/Governance.ts

```diff
@@ -481,9 +481,42 @@ export class Governance extends API {
 
   async getProposalVotes(proposalId: string) {
     const response = await this.fetch<ApiResponse<SnapshotVote[]>>(
-      `/snapshot/proposal-votes/${proposalId}`,
+      `/snapshot/votes/${proposalId}`,
       this.options().method('GET')
     )
     return response.data
   }
+
+  async getAllVotesBetweenDates(start: Date, end: Date) {
+    const response = await this.fetch<ApiResponse<SnapshotVote[]>>(
+      `/snapshot/votes/all`,
+      this.options().method('POST').json({ start, end })
+    )
+    return response.data
+  }
+
+  async getSnapshotProposals(start: Date, end: Date, fields: (keyof SnapshotProposal)[]) {
+    const response = await this.fetch<ApiResponse<Partial<SnapshotProposal>[]>>(
+      `/snapshot/proposals`,
+      this.options().method('POST').json({ start, end, fields })
+    )
+    return response.data
+  }
+
+  async getPendingProposals(start: Date, end: Date, fields: (keyof SnapshotProposal)[], limit: number) {
```

Comment:
Maybe it would be great if this receives an object. It will make it easier to read everywhere we call for this method.

E.g:
```
Governance.get().getPendingProposals({
  start,
  date,
  fields: ['id', 'total'],
  limit: 5
})
```